ic-os/components/setupos-scripts/install-guestos.sh

```diff
@@ -17,12 +17,19 @@ function install_guestos() {
     log_and_halt_installation_on_error "${?}" "Unable to activate HostOS volume group."
 
     TMPDIR=$(mktemp -d)
+    # Release RAM.  Cannot be run concurrently with install-hostos.sh.
+    rm -f disk.img
+    # Extract the disk image to RAM.
+    echo "* Temporarily extracting the GuestOS image to RAM; please stand by for a few seconds"
     tar xafS /data/guest-os.img.tar.zst -C "${TMPDIR}" disk.img
-
-    size=$(wc -c <"${TMPDIR}/disk.img")
-    size="${size:=0}"
-
-    pv -f -s "$size" "${TMPDIR}/disk.img" | dd of=${LV} bs=10M conv=sparse
+    log_and_halt_installation_on_error "${?}" "Unable to extract GuestOS disk-image."
+    # Duplicate the image to the disk.
+    # Progress is handled by status=progress.
+    # Makes a huge difference when running the setup under QEMU with no KVM.
+    # dd will detect nulls in chunks of 4M and sparsify the writes.
+    # In *non-KVM-accelerated* VM, this goes 500 MB/s, three times as fast as before.
```

Comment:
Is this comment all necessary? It's good information but do we need it in code? Especially when it's referring to "before"

Reply:
Given that the repo squashes individual commits, so I can't put this in the commit message, it's useful information for future people who need to find out why this got changed to be the way it is.